scripts/trust-keys.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+export PASSWORD_STORE_DIR=../private/passwords
```

Comment:
`.private` instead of `private`

```suggestion
export PASSWORD_STORE_DIR=../.private/passwords
```